action.yml

```diff
@@ -51,12 +51,16 @@ runs:
         JULIA_PKG_SERVER: ""
     - run: |
         # The Julia command that will be executed
-        julia_cmd=( julia --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e 'import Pkg;include(joinpath(ENV["GITHUB_ACTION_PATH"], "kwargs.jl"));kwargs = Kwargs.kwargs(;coverage = :(${{ inputs.coverage }}),force_latest_compatible_version = :(${{ inputs.force_latest_compatible_version }}), julia_args = ["--check-bounds=${{ inputs.check_bounds }}"]);Pkg.test(; kwargs...)' )
+        julia_cmd="julia --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e 'import Pkg;include(joinpath(ENV[\"GITHUB_ACTION_PATH\"], \"kwargs.jl\"));kwargs = Kwargs.kwargs(;coverage = :(${{ inputs.coverage }}),force_latest_compatible_version = :(${{ inputs.force_latest_compatible_version }}), julia_args = [\"--check-bounds=${{ inputs.check_bounds }}\"]);Pkg.test(; kwargs...)'"
```

Comment:
Sorry I forgot about this for so long.

My understanding of bash is that for commands like this, parantheses should be used. The if/then block below deals with the main issue we ran into before when using double quotes but there might be other unintentional consequences I'm not aware of. So I'd prefer to stick to the best practice that was recommended back then rather switching to double quotes.

The second suggestion in #41 seems like it should work without going back to double quotes.